SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -225,6 +227,59 @@ namespace ESpatialOSNetFlow
 	};
 }
 
+UENUM()
+namespace ESpatialOSRuntimeVariant
+{
+	enum Type
+	{
+		Standard,
+		CompatibilityMode
+	};
+}
+
+USTRUCT()
+struct SPATIALGDKEDITOR_API FRuntimeVariantVersion
+{
+	friend class USpatialGDKEditorSettings;
+	friend class FSpatialRuntimeVersionCustomization;
+
+	GENERATED_BODY()
+
+	FRuntimeVariantVersion();
+
+	FRuntimeVariantVersion(const FString& InPinnedVersion) : PinnedVersion(InPinnedVersion)
+	{}
+
+	/** Returns the Runtime version to use for cloud deployments, either the pinned one, or the user-specified one depending of the settings. */
+	const FString& GetVersionForCloud() const;
```

Comment:
Instead of GetVersionForCloud and GetVersionForLocal, can't we just have a GetVersion which internally checks if it's pinned or not?

Reply:
Not sure I follow, they do different things. The variant can have a pinned version but the cloud and local can be changed if a user desires. It's basically just a refactor of Nic's original code.

Reply:
Yes, the two methods are there to allow a user to specify a different version for the cloud and the local deployment. It's a rather advanced use case, and was introduced because the "Standard" runtime had different package names for local and cloud, but we could imagine a situation where some project needs a cloud specific fix that would deteriorate local workflow.

Reply:
Referring to this line - 
	const FString& RuntimeVersionString =
		RuntimeVersion.GetUseGDKPinnedRuntimeVersionForCloud() ? RuntimeVersion.GetPinnedVersion() : RuntimeVersion.GetVersionForCloud();
If `RuntimeVersion.GetUseGDKPinnedRuntimeVersionForCloud()` is true, why would you ever want to receive `RuntimeVersion.GetVersionForCloud()` ?

Reply:
Resolved! This was to do with the GetRuntimeVersion in the cloud deploy window :D